implementations/rust/ockam/ockam_api/src/cli_state.rs

```diff
@@ -258,6 +258,34 @@ impl IdentitiesState {
         Ok(IdentityState { path, config })
     }
 
+    pub fn list(&self) -> Result<Vec<IdentityState>> {
+        let mut identities = vec![];
+        for entry in std::fs::read_dir(&self.dir)? {
+            let entry = entry?;
```

Comment:
Maybe I'd rename `entry` to `identity` to give the reader more context as to what is what we are reading here.